lib/Hero/Content.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Button as ButtonComponent } from '../Button';
+
+const Content = ({
+  classNames = {},
+  title,
+  subTitle,
+  imageUrl,
+  hasOverlay,
+  NextImage,
+  Button = () => (
+    <ButtonComponent
+      displayText="Shop Now"
+      onClick={() => {
+        alert('You clicked me');
+      }}
+    />
+  ),
+}) => {
+  /**
+   * Next Image loader
+   * @returns {*}
+   */
+  const myLoader = () => {
+    return imageUrl;
+  };
+  return (
+    <>
+      <div
+        className={clsx('flex items-center justify-center', classNames.wrapper)}
+        style={{
+          background: NextImage === undefined ? 'url(' + imageUrl + ')' : '',
+          backgroundPosition: 'center',
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+          height: '100vh',
+          width: '100vw',
+        }}
+      >
+        {NextImage && (
+          <div
+            style={{
+              position: 'fixed',
+              height: '100vh',
+              width: '100vw',
+              overflow: 'hidden',
+              zIndex: -1,
+            }}
+          >
+            <NextImage
```

Comment:
Don't assume the custom image component will be `next/image`. Look at the way it was done in other components that have support for an `Image` prop. We simply take the `Image` component prop and render `<Image />` without passing any props to it. It's the responsibility of the developer using the library to pass the right props to their custom image component.